LookingGlass/StatsDisplay/StatsDisplayClass.cs

```diff
@@ -144,16 +148,18 @@ public void CalculateStuff()
                 if (textComponent && layoutElement)
                 {
                     textComponent.text = stats;
-                    int num = stats.Split('\n').Length;
+                    int nlines = (statsDisplayOverrideHeight.Value
+                        ? statsDisplayOverrideHeightValue.Value
+                        : stats.Split('\n').Length);
                     if (originalFontSize == -1)
                     {
                         originalFontSize = textComponent.fontSize;
                     }
                     textComponent.fontSize = statsDisplaySize.Value == -1 ? originalFontSize : statsDisplaySize.Value;
-                    layoutElement.preferredHeight = textComponent.fontSize * (num + 1);
+                    layoutElement.preferredHeight = textComponent.fontSize * (nlines + 1);
```

Comment:
As per my general comment earlier, I think it would be best to rework it so when the user doesn't specify a size (aka: automatic mod) we should just have 

`layoutElement.preferredHeight = textComponent.renderedHeight;`